### ISSUE TITLE
Batch Sync

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#467]
See: https://www.youtube.com/watch?v=jIyePcwDDyw

**Comment by randomeizer** [comment]
Yeah, I was doing this myself on a project just the other day. Worth
looking into.

The main challenge will be handling how much flexibility we give with
selecting what metadata to copy, etc. I sense a webview coming 😉

**Comment by latenitefilms** [comment]
Haha, you can never have too many webviews!

I'm going to try and throw together a little proof-of-concept over the next few days for his NAB presentation. Keep it simple for now:

* User selects all the clips in a group that need to be synced (i.e. say one video clip and two audio clips)
* CommandPost triggers "New Multicam Clip"
* CommandPost enters Multicam Clip Name based on previously selected clips
* CommandPost presses OK
* CommandPost waits until the syncing is complete
* CommandPost selects the newly created multicam clip
* CommandPost enters in the Scene Name based off Multicam Clip Name
* CommandPost enters Shot/Take based off Multicam Clip Name
* CommandPost enters Notes taken from one of the originally selected clips

If I can get this working, I might look into the ability to select multiple "groups" to process as a batch.

This proof of concept will just be a Plugin - not something for the CommandPost core, but if it works, we should definitely look to expand upon it and improve it. Probably after `1.0.0` is out in the wild.
